example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -137,23 +144,34 @@ describe('The Discover Granules workflow with https Protocol', () => {
   describe('IngestGranule workflow', () => {
     let ingestGranuleWorkflowArn;
     let ingestGranuleExecutionStatus;
+    let lambdaOutput;
 
     beforeAll(async () => {
       ingestGranuleWorkflowArn = queueGranulesOutput.payload.running[0];
       console.log('\nwait for ingestGranuleWorkflow', ingestGranuleWorkflowArn);
       ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn);
     });
 
+    afterAll(async () => {
+      await Promise.all(lambdaOutput.payload.granules.map(
+        (granule) => deleteGranule({
+          prefix: config.stackName,
+          granuleId: granule.granuleId,
+        })
+      ));
+    });
+
     it('executes successfully', () => {
       expect(ingestGranuleExecutionStatus).toEqual('SUCCEEDED');
     });
 
     describe('SyncGranule lambda function', () => {
       it('outputs 1 granule', async () => {
```

Comment:
super-minor-nit: consider updating to something like 'outputs the expected granule'

Reply:
[`815243d` (#2254)](https://github.com/nasa/cumulus/pull/2254/commits/815243d4a145724b531e1380b1140e63f738bb96)